README.md

```diff
@@ -1 +1,33 @@
-# spark-dist
\ No newline at end of file
+# Spark dist
+
+The purpose of this Spark distribution is to allow accessing S3 with AWS SAML authentication.
```

Comment:
It can be used for accessing s3 in general, and AWS SAML temporary credential in particular